src/TwigComponent/composer.json

```diff
@@ -27,21 +27,21 @@
     },
     "require": {
         "php": ">=8.1",
-        "symfony/dependency-injection": "^5.4|^6.0|^7.0",
+        "symfony/dependency-injection": "^5.4|^6.0|^7.0|^8.0",
         "symfony/deprecation-contracts": "^2.2|^3.0",
-        "symfony/event-dispatcher": "^5.4|^6.0|^7.0",
-        "symfony/property-access": "^5.4|^6.0|^7.0",
+        "symfony/event-dispatcher": "^5.4|^6.0|^7.0|^8.0",
+        "symfony/property-access": "^5.4|^6.0|^7.0|^8.0",
         "twig/twig": "^3.10.3"
     },
     "require-dev": {
-        "symfony/console": "^5.4|^6.0|^7.0",
-        "symfony/css-selector": "^5.4|^6.0|^7.0",
-        "symfony/dom-crawler": "^5.4|^6.0|^7.0",
-        "symfony/framework-bundle": "^5.4|^6.0|^7.0",
-        "symfony/phpunit-bridge": "^6.0|^7.0",
+        "symfony/console": "^5.4|^6.0|^7.0|^8.0",
+        "symfony/css-selector": "^5.4|^6.0|^7.0|^8.0",
+        "symfony/dom-crawler": "^5.4|^6.0|^7.0|^8.0",
+        "symfony/framework-bundle": "^5.4|^6.0|^7.0|^8.0",
+        "symfony/phpunit-bridge": "^6.0|^7.0|^8.0",
         "symfony/stimulus-bundle": "^2.9.1",
-        "symfony/twig-bundle": "^5.4|^6.0|^7.0",
-        "symfony/webpack-encore-bundle": "^1.15"
+        "symfony/twig-bundle": "^5.4|^6.0|^7.0|^8.0",
+        "symfony/webpack-encore-bundle": "^1.15|^2.3.0"
```

Comment:
We (dev) require WebpackEncoreBundle to ensure compability with the old `AbstractStimulusDto` class. 

This class does not exist anymore in WebpackEncoreBundle ^2, but upgrading it here allows TwigComponent to be installed with Symfony 8.0, since WebpackEncoreBundle ^1.6 does not support Symfony 8.0.

Reply:
The goal is to remove it in UX 3.x